.github/workflows/unit.yaml

```diff
@@ -13,10 +13,13 @@ jobs:
   unit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
I wanted to update it with rest of the actions in #1117, but bash script fails with v3.